app/models/piece.rb

```diff
@@ -31,60 +31,32 @@ def move_to!(x, y)
     end
   end
 
-# Returns true/false if a piece is obstructed in its movement.
-# Errors for movements not diagonal, vertical or horizontal
-  def is_obstructed?(x, y)
-  # Immediately return false if moving piece is a Knight, which is exempt from obstruction
+  # Returns true/false if a piece is obstructed in its movement.
+  def obstructed?(x, y)
+    # Immediately return false if moving piece is a Knight, which is exempt from obstruction
```

Comment:
So I did some digging and found that Rubocop is based on the Ruby style guide found here:
https://github.com/bbatsov/ruby-style-guide

In the section called **Naming** under the style guide I found this mentioned for predicates:
> Avoid prefixing predicate methods with the auxiliary verbs such as is, does, or can. These words are redundant and inconsistent with the style of boolean methods in the Ruby core library, such as empty? and include?.

That explains why I was getting that citation.

Don't know how important it is for us to follow Rubocop and The Ruby Style Guide so I went ahead and changed the method name back to _is_obstructed_. 

What are your thoughts?